zebedee-cms/src/test/java/com/github/onsdigital/zebedee/permissions/service/JWTPermissionsServiceImplTest.java

```diff
@@ -100,51 +99,59 @@ public void isPublisher_Session_NullGroup_ShouldReturnFalse() throws Exception {
     }
 
     @Test
-    public void isAdministrator_Session_Admin_ShouldReturnTrue() throws Exception {
+    public void isAdministrator_Session_ShouldError() throws Exception {
+        Session session = new Session(TEST_SESSION_ID, TEST_USER_EMAIL);
+        UnsupportedOperationException exception = assertThrows(UnsupportedOperationException.class, () ->
+                jwtPermissionsService.isAdministrator(session));
+        assertEquals("JWT sessions are enabled: isAdministrator is no longer supported", exception.getMessage());
+    }
+
+    @Test
+    public void isAdminUser_Session_Admin_ShouldReturnTrue() throws Exception {
         List<String> sessionGroups = new ArrayList<>();
         sessionGroups.add(ADMIN);
 
         Session session = new Session(TEST_SESSION_ID, TEST_USER_EMAIL, sessionGroups);
-        assertTrue(jwtPermissionsService.isAdministrator(session));
+        assertTrue(jwtPermissionsService.isAdminUser(session));
     }
 
     @Test
-    public void isAdministrator_Session_Publisher_ShouldReturnFalse() throws Exception {
+    public void isAdminUser_Session_Publisher_ShouldReturnFalse() throws Exception {
         List<String> sessionGroups = new ArrayList<>();
         sessionGroups.add(PUBLISHER);
 
         Session session = new Session(TEST_SESSION_ID, TEST_USER_EMAIL, sessionGroups);
-        assertFalse(jwtPermissionsService.isAdministrator(session));
+        assertFalse(jwtPermissionsService.isAdminUser(session));
```

Comment:
This is now a protected method and probably not worth keeping these tests here?

It seems `isAdminUser` is only present in `JWTPermissionsServiceImpl` so to be used by its child class `PermissionsServiceImpl` (from `isAdministrator`) which will go at some point. Maybe we could move it to the child class and these tests to a test class for `PermissionsServiceImpl` testing the public `isAdministrator` instead?

And do something similar for the `isPublisher` too.